src/main/java/Config/Routes/GetRoute.java

```diff
@@ -0,0 +1,12 @@
+package Config.Routes;
+
+import Config.Method;
+import Controller.Handler.Handler;
+
+public class GetRoute extends Route {
+	public GetRoute(Method method, String path, Handler handler) {
```

Comment:
Nitpick--I might capitalize the method and name this `GETRoute`, just because it took me a minute to figure out that this is the data object for the route GET, and it's not a getter for a route :)